scripts/data_overlap/ngram_hasher.py

```diff
@@ -0,0 +1,121 @@
+from typing import List, Tuple, Dict
+
+# A prime number chosen as a base for exponentiation, larger than 171,476 (estimated vocab size of English words)
```

Comment:
Given that the vocabulary is likely larger than 171,476 because of non-English text and symbols, is the choice of prime still valid?

Reply:
Yes, the number was more of a heuristic